spec/models/user_spec.rb

```diff
@@ -21,8 +37,10 @@
   end
 
   it 'sends an email' do
-    #byebug
-    expect { donor.send_instructions }
-      .to change { ActionMailer::Base.deliveries.count }.by(1)
+   # ActionMailer::Base.deliveries.clear
+    donor.send(:send_registered_email)
```

Comment:
no, i'm saying don't use the `send` method. test this wherever you like, but test it by performing the action that would trigger this method on its own